src/core/radio/radio_callbacks.cpp

```diff
@@ -58,6 +58,17 @@ void Radio::Callbacks::HandleEnergyScanDone(int8_t aMaxRssi)
     Get<Mac::SubMac>().HandleEnergyScanDone(aMaxRssi);
 }
 
+void Radio::Callbacks::HandleTransmitAckStarted(uint8_t *aFrame, uint16_t aLength)
+{
+    Mac::TxFrame frame;
+    frame.mPsdu   = aFrame;
+    frame.mLength = aLength;
+
+#if !OPENTHREAD_RADIO
```

Comment:
Remove this? Enh-ACK could be generated at RCP side so that all the encryption processing is done in `SubMac` at RCP side. RCP sim has `OT_RADIO_CAPS_TRANSMIT_SEC` set to 1, so the transmit security is handled at RCP.